caiman/tests/test_sbx.py

```diff
@@ -90,37 +91,61 @@ def test_load_subind():
     npt.assert_array_equal(data_3d_plane0_3d[:, :, :, 0], data_3d_plane0_2d)
 
 
-def test_sbx_to_tif():
-    tif_file = os.path.join(caiman_datadir(), 'temp', 'sbxtest1.tif')
+def test_load_efficiency():
+    # Make sure that when loading, excess copies are not being made and 
+    # data outside subindices are not being loaded into memory
     file_2d = os.path.join(TESTDATA_PATH, '2d_sbx.sbx')
-    data_2d_from_sbx = cm.load(file_2d)
-    sbx_utils.sbx_to_tif(file_2d, fileout=tif_file)
-    data_2d_from_tif = cm.load(tif_file)
-    npt.assert_array_almost_equal(data_2d_from_sbx, data_2d_from_tif,
-                                  err_msg='Data do not match when loaded from .sbx vs. .tif')
+    tracemalloc.start()
+    data_2d_sliced = sbx_utils.sbxread(file_2d, subindices=(slice(None), slice(None, None, 2)))
+    curr_mem, peak_mem = tracemalloc.get_traced_memory()
+    assert peak_mem / curr_mem < 1.1, 'Too much memory allocated when loading'
```

Comment:
I hope this won't lead to issues on test systems without a lot of RAM

Reply:
I think it should be fine (relative to other tests) since the same entire file is loaded elsewhere? Or did you have a different concern than just running out of RAM?

Reply:
I'm worried about a false positive, but not deeply worried; we can adapt later if we start actually seeing this in circumstances where the test becomes a burden.